test/spec/token.js

```diff
@@ -0,0 +1,40 @@
+define(function(require) {
+  var OktaAuth = require('OktaAuth');
+  var Q = require('q');
+  var tokens = require('../util/tokens');
+
+  describe('token.decode', function () {
+
+    function setup() {
+      return Q.resolve(new OktaAuth({
```

Comment:
No, but we'll be adding more functionality to the token spec that will require promises. This is also the pattern we use in all our other specs.